python/dolfinx/fem/function.py

```diff
@@ -252,12 +254,13 @@ def dtype(self) -> np.dtype:
 class Function(ufl.Coefficient):
     """A finite element function that is represented by a function space
     (domain, element and dofmap) and a vector holding the
-    degrees-of-freedom
+    degrees-of-freedom."""
 
-    """
+    _cpp_object: typing.Tuple[_cpp.fem.Function_complex64, _cpp.fem.Function_complex128,
```

Comment:
Reminder to self: 
* another class attribute vs instance attribute doubt.
* another `tuple` vs `union` doubt.